src/test/java/de/tum/in/www1/artemis/util/CourseTestService.java

```diff
@@ -1524,6 +1526,66 @@ public void searchStudentsInCourse() throws Exception {
         assertThat(tutors).isEmpty();
     }
 
+    public void searchUsersInCourse() throws Exception {
+        var course = database.createCourse();
+        // Test 1: search for all (no login or name) tutors and editors (tutors includes also editors)
+        var resultTest1 = searchUsersTest(course, List.of("tutors"), Optional.empty(), numberOfTutors + numberOfEditors, true);
+        assertThat(resultTest1.stream().filter(UserPublicInfoDTO::getIsEditor).collect(Collectors.toList())).hasSize(numberOfEditors);
```

Comment:
```suggestion
        assertThat(resultTest1.stream().filter(UserPublicInfoDTO::getIsEditor).toList()).hasSize(numberOfEditors);
```
Same for all other lists

Reply:
Since your only checking the size, you can also simply call count() on the stream itself

Reply:
Removed not needed collect in 28005df6e024ac413e2fde972f6868da8d65f438